unittests/32Bit_ASM/FEX_bugs/GOT_calculation.asm

```diff
@@ -0,0 +1,19 @@
+%ifdef CONFIG
+{
+  "RegData": {
+    "RAX": "0x10011"
+  },
+  "Mode": "32BIT"
+}
+%endif
+
+mov esp, 0xe0000010
+
+; This is a common pattern in 32-bit PIE code.
+; It uses a call + pop to calculate the GOT
+; Don't ask me why the compiler doesn't just use lea.
```

Comment:
Isn't this for the reason mentioned in the PR description?

Reply:
oop, old comment while I was investigating. Indeed.

Reply:
Comment updated.